docs/src/_reference/command-line-interface.md

```diff
@@ -9,6 +9,15 @@ Meltano provides a command line interface (CLI) that makes it easy to manage you
 To quickly find the `meltano` subcommand you're looking for, use the Table of Contents in the sidebar.
 For a better understanding of command line documentation syntax, the [docopt](http://docopt.org/) standard is useful.
 
+## Global Options
+
+The following options are available for all `meltano` subcommands:
+
+* `--enviroment` - The [Meltano Environment](/concepts/environments) to use. Defaults to `dev`. Can also be set with the `MELTANO_ENVIRONMENT` environment variable.
```

Comment:
@edgarrmondragon this isn't really true though, right? Commands like `init` and `ui` don't do anything with `--environment`. 

We have new subsections for the relevant commands on using `--environment` (see https://docs.meltano.com/reference/command-line-interface#using-add-with-environments) so I think we can remove this line.

Reply:
@tayloramurphy done: 8daebcbe15ba57f9c911b8e9e14bd768a526eca3